frontend/e2e/specs/sidePath/config-select-one-metrics.spec.ts

```diff
@@ -0,0 +1,263 @@
+import { config as metricsStepData } from '../../fixtures/createNew/metricsStep';
+import { config as configStepData } from '../../fixtures/createNew/configStep';
+import { test } from '../../fixtures/testWithExtendFixtures';
+import { clearTempDir } from '../../utils/clearTempDir';
+import { format } from '../../utils/dateTime';
+
+test.beforeAll(async () => {
+  await clearTempDir();
+});
+
+test('Only select velocity metrics on config page', async ({ homePage, configStep, metricsStep, reportStep }) => {
```

Comment:
Looks like the dupliciated test cases, you can use [test-parameterize](https://playwright.dev/docs/test-parameterize)